test/remote/gateways/remote_credorax_test.rb

```diff
@@ -79,11 +79,31 @@ def test_successful_purchase_with_auth_data_via_3ds1_fields
     assert_equal 'Succeeded', response.message
   end
 
+  def test_successful_purchase_with_auth_data_via_3ds1_fields_passing_3ds_version
+    options = @options.merge(
+      eci: '02',
+      cavv: 'jJ81HADVRtXfCBATEp01CJUAAAA=',
+      xid: '00000000000000000501',
+      # Having processor-specification enabled in Credorax test account causes 3DS tests to fail without a r1 (processor) parameter.
+      processor: 'CREDORAX',
+      three_ds_version: '1.0'
+    )
+
+    response = @gateway.purchase(@amount, @fully_auth_card, options)
+    assert_success response
+    assert_equal '1', response.params['H9']
+    assert_equal 'Succeeded', response.message
+  end
+
   def test_successful_purchase_with_3ds2_fields
     options = @options.merge(@normalized_3ds_2_options)
     response = @gateway.purchase(@amount, @three_ds_card, options)
-    assert_success response
-    assert_equal 'Succeeded', response.message
+    assert response.test?
+    refute response.authorization.blank?
+
+    refute response.params['3DS_METHOD'].blank?
+    assert_equal '2.1.0', response.params['3DS_VERSION']
+    assert_equal 'Transaction pending cardholder authentication.', response.message
```

Comment:
not sure why this worked as a direct auth before with this card...the behavior now observed seems logical (auth required for 3ds enabled card), but I'm unsure why it changed.

Reply:
I'm not sure if this is helpful or not, but in checking Credorax's documentation online, it shows that they just updated their API docs on November 26: https://epower.credorax.com/

Sadly, I don't see a changelog anywhere.

Reply:
good find, thank you!!